Author: Kees Cook <kees@ubuntu.com>
Description: extend stack buffers to be large enough for the type values
 they are expected to hold.
Bug: https://savannah.gnu.org/bugs/?32122
Bug-Ubuntu: https://launchpad.net/bugs/569802

Index: texinfo-4.13a.dfsg.1/makeinfo/sectioning.c
===================================================================
--- texinfo-4.13a.dfsg.1.orig/makeinfo/sectioning.c	2011-01-13 14:12:35.382016526 -0800
+++ texinfo-4.13a.dfsg.1/makeinfo/sectioning.c	2011-01-13 14:14:41.285127427 -0800
@@ -256,13 +256,13 @@
     return xstrdup ("");
   else if (enum_marker == APPENDIX_MAGIC)
     {
-      char s[1];
+      char s[2];
       sprintf (s, "%c", numbers[0] + 64);
       return xstrdup (s);
     }
   else
     {
-      char s[5];
+      char s[32];
       sprintf (s, "%d", numbers[0]);
       return xstrdup (s);
     }
